lib/mongery.rb

```diff
@@ -28,6 +28,10 @@ def find(*args)
       build_query.where(*args)
     end
 
+    def count
```

Comment:
this method is not tested and will break if you do so - seems this is not necessary since you can call `find().count`